bird_Docker_build/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.9
+
+WORKDIR /
+
+RUN wget https://github.com/stan-dev/cmdstan/releases/download/v2.30.0/cmdstan-2.30.0.tar.gz
+RUN tar -zxpf cmdstan-2.30.0.tar.gz
+
+RUN mv cmdstan-2.30.0 cmdstan
+WORKDIR /cmdstan
+RUN make build
+
+COPY BIRD.stan .
+RUN make BIRD
+
+COPY run_BIRD.sh run_BIRD.sh
+CMD run_BIRD.sh 
```

Comment:
run_BIRD.sh runs only graham's files. This should be a more generic BIRD docker image, so I think you should remove these lines and add an ENTRYPOINT of `["bird.py", "BIRD-1.1"]`. I think you'll have to add some more stuff to this Dockerfile to get that to work.